tests/large/conf/eventgen_replay_end_2.conf

```diff
@@ -5,6 +5,7 @@ earliest = -5s
 sampletype = raw
 outputMode = stdout
 end = -1
+interval = 0
```

Comment:
@lephino we should probably document this type of usage. Currently, our spec file lists interval as "Only valid in mode = sample".

Reply:
This is the line I'm referencing: https://github.com/splunk/eventgen/blob/SubDivideDemo/splunk_eventgen/splunk_app/README/eventgen.conf.spec#L283
Can this be updated to briefly mention usage with replay mode?